src/core/installAddon.js

```diff
@@ -333,6 +333,22 @@ export class WithInstallHelpers extends React.Component {
     }
   }
 
+  async getAddonEnabledStatus() {
+    const {
+      _addonManager,
+      addon: { guid },
+    } = this.props;
+
+    log.info('Getting add-on enabled status');
+    try {
+      const addon = await _addonManager.getAddon(guid);
+      return addon.isEnabled;
+    } catch (error) {
+      log.error(`Caught error from addonManager: ${error}`);
+      return false;
```

Comment:
Can we move the return statement outside the catch block? so that it shows that this function returns `false` by default.